tuna/update_golden.py

```diff
@@ -227,14 +252,39 @@ def merge_golden_entries(session, dbt, golden_v, entries, simple_copy=False):
   return count
 
 
+def verif_no_duplicates(golden_v, entries):
```

Comment:
since this raises a ValueError I think that should be caught in main followed by a cleaner exit.
This function need a utest, same with get_fdb_entries

Reply:
updated